src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -107,16 +109,11 @@ class BlockingProcess(
     }
 
     try {
-      val stdin =
-        context.actorOf(OutputStreamSubscriber.props(process.getOutputStream), "stdin")
+      val stdin = Sink.outputStream(process.getOutputStream)
+      val stdout = Source.inputStream(process.getInputStream)
+      val stderr = Source.inputStream(process.getErrorStream)
```

Comment:
To be honest - I'm not sure.

The Actor within the `Source.inputStream()` is created by calling `materializer.actorOf` - so my understanding this means this Actor is the child of where the Source is materialized. I'm not 100% sure if my understanding is corrrect.

Perhaps someone from Akka team can help clarify please (@2m or @ktoso) ?